src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -86,8 +92,19 @@ public void parseCommand_find() throws Exception {
 
     @Test
     public void parseCommand_help() throws Exception {
+
+        //@@author jasmoon
         assertTrue(parser.parseCommand(HelpCommand.COMMAND_WORD) instanceof HelpCommand);
-        assertTrue(parser.parseCommand(HelpCommand.COMMAND_WORD + " 3") instanceof HelpCommand);
+        assertTrue(parser.parseCommand(HelpCommand.COMMAND_ALIAS + " find") instanceof HelpCommand);
+        try {
+            parser.parseCommand(HelpCommand.COMMAND_WORD + " 3");
+            fail("The expected ParseException was not thrown");
+        } catch (ParseException pe) {
+            assertEquals("Help for '3' is unknown or not available.", pe.getMessage());
+        }
+        //need help with this test
+        //HelpCommand command = (HelpCommand) parser.parseCommand(HelpCommand.COMMAND_WORD + " find");
+        //assertEquals(new HelpCommand("find").execute(), command.execute());
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.